tools/jsii-compliance/schema.ts

```diff
@@ -72,11 +72,6 @@ export interface BindingExclusion {
  * Exclusion of a specific test from a specific binding.
  */
 export interface TestExclusion {
-  /**
-   * The language to exclude the test from.
-   */
-  readonly language: string;
```

Comment:
Just an oversight, doesn't belong here since this is they key of the exclusions map.